nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.playerctld;
+in
+{
+  options.services.playerctld = {
+
+    enable = mkEnableOption (lib.mdDoc "Enable the playerctld daemon.");
+
+    package = mkPackageOption pkgs "playerctl" {
+      default = [ "playerctl" ];
+    };
+
+  };
+
+  config = mkIf cfg.enable {
+
+    environment.systemPackages = with pkgs; [ cfg.package ];
+
+    systemd.user.services.playerctld = {
+      description = "Playerctld daemon to track media player activity";
+      wantedBy = [ "default.target" ];
+
+      serviceConfig = {
+        Type = "exec";
+        ExecStart = "${cfg.package}/bin/playerctld";
+      };
```

Comment:
Would some hardening make sense here? Something like `DynamicUser=true` or `PrivateTmp=true` ? Because without a `User=` specification it would run as root, which should be avoided.